src/ontology/cl-edit.owl

```diff
@@ -31266,6 +31267,16 @@ AnnotationAssertion(rdfs:label obo:CL_4033036 "OFFx cell")
 SubClassOf(obo:CL_4033036 obo:CL_0000750)
 SubClassOf(obo:CL_4033036 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0001786))
 
+# Class: obo:CL_4033037 (tracheobronchial mucus secreting glandular cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:30864819") obo:IAO_0000115 obo:CL_4033037 "A mucus secreting cell of a submucosal gland of the tracheobronchial tree.")
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4033037 <https://orcid.org/0000-0001-6677-8489>)
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4033037 "2023-04-20T10:55:50Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:30864819") oboInOwl:hasExactSynonym obo:CL_4033037 "mucus secreting cell of tracheobronchial tree submucosal gland")
+AnnotationAssertion(rdfs:label obo:CL_4033037 "tracheobronchial mucus secreting glandular cell")
+EquivalentClasses(obo:CL_4033037 ObjectIntersectionOf(obo:CL_0000319 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0007196) ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_8410077)))
```

Comment:
Same inferences result by removing `and ('part of' some 'tracheobronchial tree')`, so consider if it is necessary to include in the logical definition.

Reply:
I want to maintain it, as there are also submucosal glands in upper airways. In case they are added, mucus secreting cells in those glands would be inferred under mucus secreting cell of tracheobronchial tree submucosal gland.